editoast/src/views/layers/mod.rs

```diff
@@ -56,19 +62,53 @@ impl LayersError {
     }
 }
 
-#[derive(Deserialize, Debug, Clone)]
+#[derive(Deserialize, Debug, Clone, IntoParams)]
+#[into_params(parameter_in = Query)]
 struct InfraQueryParam {
     infra: i64,
 }
 
+#[derive(Deserialize, IntoParams)]
+#[allow(unused)]
+struct LayerViewParams {
+    layer_slug: String,
+    view_slug: String,
+}
+
+#[derive(Serialize, ToSchema)]
+struct ViewMetadata {
+    #[serde(rename = "type")]
+    data_type: String,
+    #[schema(example = "track_sections")]
+    name: String,
+    #[serde(rename = "promoteId")]
+    #[schema(example = json!({track_sections: "id"}))]
+    promote_id: JsonValue,
+    #[schema(example = "xyz")]
+    scheme: String,
+    #[schema(example = json!(["http://localhost:7070/tile/track_sections/geo/{z}/{x}/{y}/?infra=1"]))]
+    tiles: Vec<String>,
+    attribution: String,
+    minzoom: u64,
+    #[schema(example = 15)]
+    maxzoom: u64,
+}
+
 /// Returns layer view metadata to query tiles
-#[get("/layer/{layer_slug}/mvt/{view_slug}")]
+#[utoipa::path(
+    tag = "layers",
+    params(InfraQueryParam, LayerViewParams),
+    responses(
+        (status = 200, body = ViewMetadata, description = "Successful Response"),
```

Comment:
```suggestion
        (status = 200, body = inline(ViewMetadata), description = "Successful Response"),
```

Since it's only used here you can inline the schema, that way you won't have to propagate it up to views/mod using schema!